easybuild/easyconfigs/s/Scoary/Scoary-1.6.16-foss-2021a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'Scoary'
+version = '1.6.16'
+
+homepage = 'https://github.com/AdmiralenOla/Scoary'
+description = "Microbial pan-GWAS using the output from Roary"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('Tkinter', '%(pyver)s'),
+    ('ETE', '3.1.2'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('argparse', '1.4.0', {
+        'checksums': ['62b089a55be1d8949cd2bc7e0df0bddb9e028faefc8c32038cc84862aefdd6e4'],
+    }),
+    (name, version, {
+        'source_tmpl': 'scoary-%(version)s.tar.gz',
+        'checksums': ['f433343422c5805a70a20f7fe83851490ae3cdf07745c65680de5feea40a6dca'],
+        'modulename': False,
```

Comment:
@verdurin why set it to False? maybe set it to `scoary.methods` ?

Reply:
@smoors fixed, and `sanity_check_command` added.